ceaos/sensors/nwsensor.py

```diff
@@ -0,0 +1,26 @@
+from .sensor_definition import Sensor
+from ..loggers.InfluxDB import InfluxDBLogger
+
+
+class Network_Sensor(Sensor):
+    def __init__(self, sensor=None, ):
+        self.curr_value = None
+        self.logger = InfluxDBLogger()
+        # I think each Network_Sensor is essentially
+        # attached to an actual sensor
+        self.sensor = sensor
```

Comment:
You don't need this! Remember that the interface for a Sensor when initialized will create a NotImplementedError

Reply:
Not sure I follow, for each network sensor, don't we need to pass in an actual sensor hence, self.sensor = sensor?

Reply:
Nope, There's no need for this, we can just treat them all the same. We may want to keep track of the datatype, but that's it.